src/logic/wallets/store/actions/index.ts

```diff
@@ -1,6 +1,6 @@
-export * from './addProvider'
-export * from './fetchProvider'
-export * from './removeProvider'
-export { default as addProvider } from './addProvider'
-export { default as fetchProvider } from './fetchProvider'
-export { default as removeProvider } from './removeProvider'
+export enum PROVIDER_ACTIONS {
+  WALLET = 'provider/walletUpdated',
+  ACCOUNT = 'provider/accountUpdated',
+  NETWORK = 'provider/networkUpdated',
+  ENS = 'provider/ensUpdated',
```

Comment:
Should the actions be uppercased for consistency with other actions? i.e. `PROVIDER_ENS_UPDATED`

Reply:
The Redux style guide says [you should write action types as `domain/eventName`](https://redux.js.org/style-guide/style-guide#write-action-types-as-domaineventname).